### PR TITLE
maint: fix spelling errors

### PR DESCRIPTION
**Pull request by jsoref**
----

### Before the change?

* typos https://github.com/jsoref/webhooks/actions/runs/7950168791/attempts/1#summary-21702175342

### After the change?

* https://github.com/jsoref/webhooks/actions/runs/7950168938/attempts/1#summary-21702175695

### Pull request checklist
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

### Does this introduce a breaking change?

Please see our docs on [breaking changes](https://github.com/octokit/.github/blob/master/community/breaking_changes.md) to help!

- [ ] Yes
- [x] No

----

### CODE REVIEW

File: payload-examples/api.github.com/discussion/edited.with-reactions.payload.json
Comment (jsoref) [inline payload-examples/api.github.com/discussion/edited.with-reactions.payload.json:1]: these files don't appear to be referenced?

File: payload-examples/api.github.com/index.json [hunk 1]
```diff
@@ -56347,7 +56347,7 @@
             "type": "User",
             "site_admin": false
           },
-          "body": "Maybe you should use more emojji on this line.",
+          "body": "Maybe you should use more emoji on this line.",
```
Comment (jsoref) [inline payload-examples/api.github.com/index.json:56350]: technically this really misspelled in github, but as this is a mock, i don't think it has to match the real world.

File: payload-examples/api.github.com/package/published.npm.payload.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "package": {
     "id": 10696,
     "name": "hello-world-npm",
-    "namespace": "Codertocaat/hello-world-npm",
+    "namespace": "Codertocat/hello-world-npm",
```
Comment (jsoref) [inline payload-examples/api.github.com/package/published.npm.payload.json:6]: This feels like a typo -- at least I can't find it in npm ...

Comment (wolfy1339) [reply payload-examples/api.github.com/package/published.npm.payload.json:6]: It's just an example payload. The package doesn't necessarily exist on npm

**Comment by wolfy1339** [comment]
The `cache/` files are taken from GitHub's docs, and thus we don't control the spelling errors. I wouldn't worry too much about them

File: payload-examples/api.github.com/discussion/edited.with-reactions.payload.json
Comment (wolfy1339) [reply payload-examples/api.github.com/discussion/edited.with-reactions.payload.json:1]: They are used to test the schemas for any regressions. They are used with the `validate:payloads` script

**Review by wolfy1339 — APPROVED** [review]

**Pull request merged**